Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.h

```diff
@@ -172,6 +173,11 @@ namespace AzToolsFramework
 
         static bool ShouldReplaceRPE();
 
+        AZStd::vector<size_t> GetPathToRoot(DPERowWidget* row) const;
+
+        bool IsRecursiveExpansionOngoing();
```

Comment:
Nitpick: Could make this method `const`

```suggestion
        bool IsRecursiveExpansionOngoing() const;
```

Reply:
Done. Will push with the next commit. Waiting to see if Alex or anyone has more feedback.